### ISSUE TITLE
Missing author

### ISSUE THREAD

**Issue by lidel** [ipfs/ipfs-blog#321]
@stan-pl   I'm lacking context here, mind filling https://github.com/ipfs/ipfs-blog/blame/main/src/_blog/decentralizing-the-internet-s-root.md#L5 and https://github.com/ipfs/ipfs-blog/blame/main/src/_blog/ion-a-path-to-decentralized-identity.md#L5 with appropriate names (people / companies / projects)? :pray:

**Comment by stan-pl** [comment]
Sorry, do you mean you need the author names of this blog post? https://blog.ipfs.io/decentralizing-the-internet-s-root/

**Comment by lidel** [comment]
Correct. Same ask for https://blog.ipfs.io/ion-a-path-to-decentralized-identity/
